cogs5e/models/sheet/player.py

```diff
@@ -196,6 +211,25 @@ def get_reset_to(self):
     def value(self):
         return self._value
 
+    @property
```

Comment:
I'm not entirely sure these properties (and similarly, `_title` and `_desc` being private attrs) are entirely necessary.

As is, `CustomCounter.title` (the property here) is what is called when it's committed to the database, so the `[cname]` and `[name]`s are lost - I don't believe `[cname]` and `[name]` are necessary, though.